notebooks/tou/hpwh_schedule_basic.qmd

```diff
@@ -0,0 +1,175 @@
+  - **Note:** Future implementation will allow the agent to specify a more complicated schedule to optimize between comfort and cost. For now, assume the agent can only choose between default schedule  and TOU-adaptive schedule (only runs during off-peak hours).
+- **Simulate:** Run OCHRE with TOU-adaptive HPWH schedule to get $E_{TOU}$, $C_{bill,TOU}$.
+
+## Step 3: Anticipated Savings
+
+- **Calculate:**
+$$
+\Delta C = C_{bill,default} - C_{bill,TOU}
+$$
+- **Net Benefit:**
+$$
+\text{Net Benefit} = \Delta C - C_{switch} - C_{comfort}
+$$
+
+**Note:** The only difference in monthly bill is due to the difference in HPWH operation between the default and TOU-adaptive schedules since we are assuming all other factors remain constant in the OCHRE simulation. Thus the agent can use the difference in monthly bill to calculate the anticipated savings.
+
+## Step 4: Decision Rule
+
+- **If** $\text{Net Benefit} > 0$: Consumer switches to TOU-adapted schedule.
+- **Else:** Remains on default schedule.
+
+## Step 5: Feedback & Iteration
+
+- At each monthly billing cycle, repeat Steps 1–4 using updated consumption and bills.
+- Consumer can reconsider switching if conditions change (e.g., rate structure, switching cost).
+
+# 4. Integration with OCHRE
+
+## Inputs Required
+
+- Hot water usage schedule: $U_{HW}$ (from input schedule file)
+- Usage multiplier: $\lambda$ (from building HPXML file)
+- TOU rate definition: $r_{on}, r_{off}$ (passed to cost calculation)
+- HPWH scheduling control: ability to enforce on/off operation windows (via `update_external_control()` in HPWH code)
+
+## OCHRE Outputs Used
+
+| Output Name                   | Use in Model                         |
+|-------------------------------|--------------------------------------|
+| Hot Water Delivered (L/min)   | For checking comfort/unmet demand    |
+| Water Heating Delivered (W)   | For cost calculation                 |
+| Hot Water Outlet Temperature  | For comfort check (optional)         |
+| Hot Water Unmet Demand (kW)   | To quantify service/comfort loss     |
+
+## Implementation Notes
+
+- **Control:** Implement schedule-based control in OCHRE (see `update_external_control()` in HPWH code).
+- **Comfort Check:** If TOU schedule leads to unmet demand or low temp, can factor "comfort penalty" into switching cost.
+- **Outputs:** Compare cost and comfort metrics for both schedules.
+
+# 5. Example Calculation
+
+Suppose:
+
+- Default schedule: HPWH runs as needed.
+- TOU schedule: HPWH only runs 10PM–6AM (off-peak).
+- Monthly bill (default): $\$30$
+- Monthly bill (TOU): $\$22$
+- Switching cost: $\$5$ (one-time or amortized)
+- Comfort penalty: $\$2$ (due to unmet demand)
+
+$$
+\Delta C = 30 - 22 = \$8
+$$
+$$
+\text{Net Benefit} = 8 - 5 - 2 = \$1
+$$
+
+**Decision:** Consumer switches to TOU-adaptive schedule.
+
+# 6. Equations
+
+## Electricity Cost Calculation
+
+$$
+C_{bill} = \sum_{t} E(t) \cdot r(t)
```

Comment:
@sherryzuo @alexhyunminlee notation should be sum over hours in the month, right?